lib/cmd/fh3/secureendpoints/auditlog.js

```diff
@@ -0,0 +1,59 @@
+/* globals i18n */
+var common = require("../../../common");
+var fhreq = require("../../../utils/request");
+var Table = require('cli-table');
+var _ = require('underscore');
+
+module.exports = {
+  'desc' : i18n._('Audit logs from secure endpoints'),
+  'examples' :
+    [{
+      cmd : 'fhc secureendpoints auditlog --app=<app-id> --env=<environment>',
+      desc : i18n._('List all audit logs from a secure endpoints from <app> into <environment>')
+    }],
+  'demand' : ['app','env'],
+  'alias' : {
+    'app' : 'a',
+    'env' : 'e',
+    'json' : 'j',
+    0 : 'app',
+    1 : 'env',
+    2 : 'json'
+  },
+  'describe' : {
+    'app' : i18n._('Unique 24 character GUID of the application'),
+    'env' : i18n._("The environment where the app is deployed"),
+    'json' : i18n._('Output into json format')
+  },
+  'customCmd' : function(params,cb) {
+    var payload = {appId: params.app, environment: params.env};
+    common.doApiCall(fhreq.getFeedHenryUrl(), "/box/srv/1.1/app/endpointsecurity/auditLog", payload, i18n._("Error getting auditlog: "), function(err, log) {
+      if (err) {
+        return cb(err);
+      }
+
+      if (!params.json) {
+        log._table = createTableForAuditLog(log.list);
+      }
+      return cb(null, log);
+    });
+  }
+};
+
+/**
+ * Create a table out put for audit logs
+ * @param entries
+ * @returns {Table}
+ */
+function createTableForAuditLog(entries) {
```

Comment:
Sorry @wei-lee, I'd like to rectify. 

- The createTableForAuditLog already exists before. 
- I think it maybe was created for two reasons. It should show the date columns as the example 5 days ago, 
- I wanted to mean into the last comment that if we have the need to print the result as a table, and the same table will be printed into more than one command and we need to change it I think is better keep it centralized to be maintained easier since don't will be required change more than one place in the code and we will don't have duplicate code. 

Please, let me know if you prefer it made by the generic method or that we change the date column to work as the default behavior into the FHC which is for example "1 day ago".